hawc/apps/epiv2/forms.py

```diff
@@ -31,22 +39,201 @@ def helper(self):
                 widget.attrs["rows"] = 3
 
         if self.instance.id:
-            inputs = {
-                "legend_text": f"Update {self.instance}",
-                "help_text": self.UPDATE_HELP_TEXT,
-                "cancel_url": self.instance.get_absolute_url(),
-            }
+            helper = BaseFormHelper(self)
+            helper.form_tag = False
+
         else:
             inputs = {
                 "legend_text": self.CREATE_LEGEND,
                 "help_text": self.CREATE_HELP_TEXT,
                 "cancel_url": self.instance.study.get_absolute_url(),
+                "submit_text": "Next",
             }
-
-        helper = BaseFormHelper(self, **inputs)
+            helper = BaseFormHelper(self, **inputs)
 
         helper.add_row("study_design", 3, "col-md-4")
         helper.add_row("age_description", 3, "col-md-4")
         helper.add_row("summary", 2, "col-md-6")
         helper.add_row("countries", 4, "col-md-3")
+
+        return helper
+
+
+class CriteriaForm(forms.ModelForm):
+    class Meta:
+        model = models.Criteria
+        exclude = ("design",)
+        labels = {}
+
+    def __init__(self, *args, **kwargs):
+        design = kwargs.pop("parent", None)
+        super().__init__(*args, **kwargs)
+        if design:
+            self.instance.design = design
+
+    @property
+    def helper(self):
+        for fld in list(self.fields.keys()):
+            widget = self.fields[fld].widget
+            if type(widget) == forms.Textarea:
+                widget.attrs["rows"] = 3
+        helper = BaseFormHelper(self)
+        helper.form_tag = False
+        helper.add_row("name", 1, "col-md-4")
+        return helper
+
+
+class ChemicalForm(forms.ModelForm):
+    class Meta:
+        model = models.Chemical
+        exclude = ("design",)
+        labels = {}
+
+    def __init__(self, *args, **kwargs):
+        design = kwargs.pop("parent", None)
+        super().__init__(*args, **kwargs)
+        if design:
+            self.instance.design = design
+
+        self.fields["dsstox"].widget = selectable.AutoCompleteSelectWidget(
+            lookup_class=DssToxIdLookup
+        )
+
+    @property
+    def helper(self):
+        for fld in list(self.fields.keys()):
```

Comment:
I know this is a pattern in other forms that are in hawc, but i'm now thinking we should be specific where possible. can we change this to specific fields instead? Just in epiv2 for now..

```python
self.fields['field_name'].widget.attrs['rows'] = 3
```